Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -2235,7 +2235,7 @@ void Document::dispatch_events_for_animation_if_necessary(GC::Ref<Animations::An
                     {
                         { .bubbles = true },
                         css_animation.id(),
-                        elapsed_time,
+                        elapsed_time / 1000, // Convert to seconds.
```

Comment:
Instead of the inline comment, you could change the variable name here to indicate that it represents milliseconds: `elapsed_time_ms`, or similar.